### ISSUE TITLE
Load polling with idiomorph

### ISSUE THREAD

**Issue by davegaeddert** [bigskysoftware/htmx#1671]
I tried to make a codepen to show this, but couldn't figure out how. Basically, I have a div using "load polling" and it works just fine, but when I add `hx-swap="morph"` then the delay/poll fires the first time but not after that. Does that make sense?

Using htmx 1.9.4 and idiomorph 0.0.8.

```diff
 <div
     hx-get
     hx-trigger="load delay:1s"
+    hx-swap="morph"
 ></div>
```

**Comment by davegaeddert** [comment]
I ended up using regular polling ("every 3s") instead which works fine, so feel free to close this if it's not worth digging into.

**Comment by alexpetros** [comment]
Do you have the idiomorph extension loaded?

**Comment by davegaeddert** [comment]
Yeah, the morph was working fine on the first re-load. But it wouldn't fire again after that.

**Comment by ioplker** [comment]
Hi there! I have the same issue with htmx 2.0.4 (from https://unpkg.com/htmx.org@2.0.4/dist/htmx.min.js) and idiomorph-ext 0.7.3 (from https://unpkg.com/idiomorph@0.7.3/dist/idiomorph-ext.min.js).

Wanted to share a quick'n'dumb workaround (*but the `load` value would be better though*).
I have a page with paginated results. If the first (*latest in time*) page is opened server returns an invisible element with `hx-trigger="every 2s"` (*and other attrs for swap*). On other pages this element is absent, so I don't get constant refreshes when looking at some "historical" data.

I've tried to change attrs alone, but polling does not stop if the original element is still there.
Hope this helps someone.

**Comment by 1cg** [comment]
no longer relevant
